cmd/mps-control-daemon/mps/daemon.go

```diff
@@ -124,25 +127,51 @@ func (d *Daemon) Start() error {
 	}
 	defer statusFile.Close()
 
+	klog.InfoS("Starting log tailer", "resource", d.rm.Resource())
+	tail := exec.Command("tail", "-n", "+1", "-f", filepath.Join(logDir, "control.log")) //nolint:gosec
+	tail.Stdout = os.Stdout
+	if err := tail.Start(); err != nil {
+		klog.ErrorS(err, "Could not start tail command on control.log; ignoring logs")
+		d.tail = nil
+	} else {
+		d.tail = tail
+	}
+
 	return nil
 }
 
 // Stop ensures that the MPS daemon is quit.
 func (d *Daemon) Stop() error {
-	output, err := d.EchoPipeToControl("quit")
+	_, err := d.EchoPipeToControl("quit")
 	if err != nil {
 		return fmt.Errorf("error sending quit message: %w", err)
 	}
-	klog.InfoS("Shut down MPS", "output", output)
+	klog.InfoS("Stopped MPS control daemon", "resource", d.rm.Resource())
+
+	if d.tail != nil {
+		klog.InfoS("Stopping log tailer", "resource", d.rm.Resource())
+		if err := d.tail.Process.Signal(os.Kill); err != nil {
```

Comment:
Do we want os.Kill or os.Term?

Reply:
`os.Term` is probably better as it allows `tail` to clean things up.

Does it make sense to send `os.Term` and then `os.Kill` if that fails?

Reply:
That does sound cleaner. Let me update.